code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -99,9 +99,12 @@
 /obj/structure/constructshell/attackby(obj/item/O, mob/user, params)
 	if(istype(O, /obj/item/device/soulstone))
 		var/obj/item/device/soulstone/SS = O
+		if(!iscultist(user) && !iswizard(user))
```

Comment:
99% of the time when people get popped into a juggernaut they think HEY IM AN ANTAG LETS MUUUURDER. 

99% of the time when people see a juggernaut they think HEY AN ANTAG LETS MUUUURDER IT. 

99% of the time the cult sees a juggernaut they think HEY AN ALLY LETS NOT MUUUURDER IT.... OH GOD WHY IS IT MUUUURDERING US, AHELP ADMINS PLZ BAN THIS ROGUE JUGG.

I tested it, creating a juggernaut as a non-cultists doesn't even tell them who they're loyal to. They have to guess. Until someone wants to create a better system, allowing people to make rogue juggernauts just isn't worth it.